spec/classes/foreman_proxy__plugin__dns__infoblox_spec.rb

```diff
@@ -26,6 +26,7 @@
           verify_exact_contents(catalogue, '/etc/foreman-proxy/settings.d/dns_infoblox.yml', [
             '---',
             ':dns_server: "localhost"',
+            ':timeout: 60',
```

Comment:
This doesn't match the template
```suggestion
            ':dns_timeout: 60',
```

Reply:
Thanks, amended.

Reply:
There are no test adjustments to check anything regarding this parameter...